src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,16 @@
-const fizzBuzz = (number) => {};
+const fizzBuzz = (number) => {
+    if (number % 3 == 0 && number % 5 == 0) {
```

Comment:
Always use === instead of == as default, to check the type and the value as well.